bizdemo/easy_note/note/handler.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2021 CloudWeGo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package main
+
+import (
+	"context"
+
+	"github.com/cloudwego/kitex-examples/bizdemo/easy_note/note/errno"
+	"github.com/cloudwego/kitex-examples/bizdemo/easy_note/note/kitex_gen/kitex/demo/note"
+	"github.com/cloudwego/kitex-examples/bizdemo/easy_note/note/service"
+)
+
+// NoteServiceImpl implements the last service interface defined in the IDL.
+type NoteServiceImpl struct{}
+
+// CreateNote implements the NoteServiceImpl interface.
+func (s *NoteServiceImpl) CreateNote(ctx context.Context, req *note.CreateNoteRequest) (resp *note.CreateNoteResponse, err error) {
+	// TODO: Your code here...
```

Comment:
See many TODO left there. Are you forget to delete ?

Reply:
sorry, I forget to delete it. :anguished: